docker/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM debian:8-slim
+
+RUN apt-get update \
+    && apt-get install autoconf libtool nasm libpng-dev automake pkg-config build-essential wget \
+    -yq --no-install-suggests --no-install-recommends --force-yes
+
+WORKDIR /src
+RUN wget --no-check-certificate https://github.com/mozilla/mozjpeg/releases/download/v3.2/mozjpeg-3.2-release-source.tar.gz
```

Comment:
Can you target the latest release? https://github.com/mozilla/mozjpeg/releases/tag/v3.3.1